test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/DicomTagsManager.cs

```diff
@@ -57,14 +57,38 @@ public async Task<OperationStatus> AddTagsAsync(IEnumerable<AddExtendedQueryTagE
             }
 
             var operation = await response.GetValueAsync();
-
-            var operationStatus = await GetOperationStatusRetryPolicy.ExecuteAsync(async () =>
+            return await GetOperationStatusRetryPolicy.ExecuteAsync(async () =>
             {
                 var operationStatus = await _dicomWebClient.GetOperationStatusAsync(operation.Id);
                 return await operationStatus.GetValueAsync();
             });
-            return operationStatus;
         }
 
+        public async Task<GetExtendedQueryTagEntry> GetTagAsync(string tagPath, CancellationToken cancellationToken = default)
```

Comment:
I think "manager" is ok because the addition and retrieval of tags is performed through this object. And in fact, I like having these helper methods, because they help format the response for tests haha